### PR TITLE
chore(DDI-767): React Component test cases for Cypress Test

### PR DESCRIPTION
**Pull request by bluekurinji**

### CODE REVIEW

File: apps/react-demo/src/routes/radio.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      </GoARadioGroup >
+
+      <h3>Orientation</h3>
+      <GoARadioGroup name="color" value="orange" orientation="horizontal" onChange={onChange}>
+        <GoARadioItem name="color" value="red"></GoARadioItem>
+        <GoARadioItem name="color" value="blue"></GoARadioItem>
+        <GoARadioItem name="color" value="orange"></GoARadioItem>
+      </GoARadioGroup >
+
+    </>
```
Comment (chrisolsen) [inline apps/react-demo/src/routes/radio.tsx:47]: These are all using the same `name`, which will cause binding problems.

Comment (bluekurinji) [reply apps/react-demo/src/routes/radio.tsx]: fixed

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**